### ISSUE TITLE
pile_full, packed_full and scenes_tdsf_dep-nor generation scripts for 100 scenes are not present to train the GraspNeRF network 

### ISSUE THREAD

**Issue by Kishore8949** [PKU-EPIC/GraspNeRF#12]
Hi,

Thanks for your wonderful work and sharing the code.

The training script for graspnerf might be missing src/vgn, render_packed_STD_rand.py,  scenes_tsdf_dep-nor_packed.py and scenes_tsdf_dep-nor_pile.py.

The loss convergence for the training script is not able to happen and giving poor results.

Appreciate your help!

**Comment by elenacliu** [comment]
I have the same question

**Comment by chenming0011112** [comment]
the same
